packages/layout/src/link-box.tsx

```diff
@@ -54,10 +54,10 @@ export const LinkBox = forwardRef<LinkBoxProps, "div">((props, ref) => {
       {...rest}
       className={cx("chakra-linkbox", className)}
       __css={{
-        /* Elevate the links and abbreviations up */
-        "a[href]:not(.chakra-linkbox__overlay), abbr[title]": {
+        /* Elevate the links, abbreviations and buttons up */
+        "a[href]:not(.chakra-linkbox__overlay), abbr[title], button": {
           position: "relative",
-          zIndex: 1,
+          zIndex: 2,
```

Comment:
Did you encounter any issues with `zIndex: 1`?
I think `zIndex: 1` should suffice, because the `::before` element will always be _before_ any other child DOM element. Trailing DOM elements with `zIndex: 1` should be on top.

Reply:
Aaah, that makes sense. I did not figured out why `zIndex: 1` worked as expected so I set it to 2 to avoid any other unexpected behavior with other browsers/browser version.

I force-pushed it.